templates/layouts/_base.html

```diff
@@ -32,22 +32,44 @@
   <input id="csrf_token" name="csrf_token" type="hidden" value="{{ csrf_token() }}"/>
 {% endblock %}
 
-{% block scripts %}
-  {% if analytics_ua_id  %}
+
+
+{% if google_tag_manager_id and google_tag_manager_auth and google_tag_manager_preview  %}
+
+  {% block head %}
```

Comment:
For now the this `head` and `script` block below shouldn't be depended on the gtm stuff, but only made it like this since currently only things in the blocks are gtm related and to avoid 2 ifs.